docs/lib/geo/fragments/js/maps.md

```diff
@@ -0,0 +1,126 @@
+> Prerequisite: [Install and configure](~/cli/start/install.md) the Amplify CLI
+
+## Provision a map resource
+
+The primary way to provision Geo map resources is through the Amplify CLI.
+
+Note: You will need the `@geo` release of the CLI for this. For more information, please see [Getting started](~/lib/geo/getting-started.md)
+
+```bash
+amplify add geo
+```
+
+```
+? Select which capability you want to add: (Use arrow keys)
+❯ Map (visualize the geospatial data)
+  Location search (search by places, addresses, coordinates)
+```
+
+From here you can follow the prompts to generate your new Map resource. Make sure to push your changes up after configuration:
+
+```
+amplify push
+```
+
+For more information, you can visit the full [Amplify CLI Geo Maps docs](~/cli/geo/maps.md).
+
+
+## Display a map
+
+Add maplibre-gl-js to your app with `yarn` or `npm`:
+
+```bash
+npm install -S maplibre-gl
+```
+[MapLibre GL](https://github.com/maplibre/maplibre-gl-js) is an open source map rendering library for javascript.
+
+Add maplibre-gl-js-amplify to your app with `yarn` or `npm`:
+
+```bash
+npm install -S maplibre-gl-js-amplify
+```
+[maplibre-gl-js-amplify](https://github.com/aws-amplify/maplibre-gl-js-amplify) is a library that makes it easy to integration MapLibre with Amplify Geo.
+
+In your app’s entry point i.e. `App.js`, import and load the configuration file `aws-exports.js` which has been created and replaced into `/src` folder in the previous step.
```

Comment:
Next, configure the Amplify library in your app. All geo-related configuration information is available within the `aws-exports.js` file. Add the following to your app's entry point (i.e. `App.js`):

Reply:
Updated.